docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -559,7 +559,8 @@ the User guide.
 
 !!! attention
     Because SSL Passthrough works on layer 4 of the OSI model (TCP) and not on the layer 7 (HTTP), using SSL Passthrough
-    invalidates all the other annotations set on an Ingress object.
+    invalidates all the other annotations set on an Ingress object except the `backend-protocol`.
```

Comment:
@cyril-corbon this is not true. If you set `ssl-passthrough: "true"` it just pipes the connection, there is no concept of HTTP or HTTPS, only a TCP connection.

https://github.com/kubernetes/ingress-nginx/blob/a268ec493c6a1d81d0932f9aeedd1781df2fe7b1/internal/ingress/controller/tcp.go

Reply:
@cyril-corbon the proxy only copy bytes https://github.com/kubernetes/ingress-nginx/blob/a268ec493c6a1d81d0932f9aeedd1781df2fe7b1/internal/ingress/controller/tcp.go#L114, is not aware of the details.